app/jobs/subscription_payment_job.rb

```diff
@@ -26,7 +26,11 @@ def perform(subscription)
 
   def create_charge(subscription)
     user = subscription.user
-    customer = find_stripe_customer(user)
+    customer = user.find_stripe_customer
+
+    unless customer
+      # this shouldn't happen, but we need to handle this case
```

Comment:
If we need this to be handle do we have a ticket for this?

Reply:
I'll add some handling for this